traits/src/lib.rs

```diff
@@ -39,7 +39,13 @@ pub trait ExtractRecipient {
 	fn extract_recipient(dest: &MultiLocation) -> Option<Vec<u8>>;
 }
 
-pub trait FeeHandler: Sized {
+pub trait FeeHandler {
 	// Return fee represent by a specific asset
-	fn get_fee(asset_id: &AssetId) -> Option<u128>;
+	fn get_fee(domain: DomainID, asset: &AssetId) -> Option<u128>;
+}
+
+impl FeeHandler for () {
+	fn get_fee(_domain: DomainID, _asset: &AssetId) -> Option<u128> {
+		None
+	}
```

Comment:
I have some trouble to understand here. why do we need an impl of `FeeHandler::get_fee`` here but the body is None

Reply:
A little bit tricky, lol. It's some kind of an `empty` thing, very useful when we don't actually have one thing but we just want put something there. For example, we configure a dynamic fee handler in mock runtime [here](https://github.com/sygmaprotocol/sygma-substrate-pallets/blob/0f5676fc579e33388e7f849ce485c3f8596b1d12/fee-handler-router/src/mock.rs#L152)

Reply:
ahh, I see, because we dont have `DynamicFeeHandler` pallet yet but type `DynamicFeeHandler` defined in the fee handler router needs to impl FeeHandler trait, so that we need to impl a temp `get_fee` on `()`, correct me if im wrong @tolak

Reply:
Correct